File: brain/a5c781d0-76af-4561-8313-71ab1c0a2542/scratch/convert_sqlite.py
import re

schema_path = r"C:\Users\UMANG\.gemini\antigravity-ide\scratch\luxury-jewelry-platform\backend\prisma\schema.prisma"

with open(schema_path, "r", encoding="utf-8") as f:
    content = f.read()

# Replace provider
content = content.replace('provider = "mysql"', 'provider = "sqlite"')

# Remove @db.* annotations (e.g. @db.Text, @db.Decimal(12, 2), @db.VarChar(100))
content = re.sub(r'@db\.[a-zA-Z]+(?:\([^)]*\))?', '', content)

with open(schema_path, "w", encoding="utf-8") as f:
    f.write(content)

print("Schema converted to SQLite successfully!")

File: brain/a5c781d0-76af-4561-8313-71ab1c0a2542/scratch/convert_sqlite.js
const fs = require('fs');
const schemaPath = "C:\\Users\\UMANG\\.gemini\\antigravity-ide\\scratch\\luxury-jewelry-platform\\backend\\prisma\\schema.prisma";

let content = fs.readFileSync(schemaPath, 'utf8');

// Replace provider
content = content.replace('provider = "mysql"', 'provider = "sqlite"');

// Remove @db.* annotations (e.g. @db.Text, @db.Decimal(12, 2), @db.VarChar(100))
content = content.replace(/@db\.[a-zA-Z]+(?:\([^)]*\))?/g, '');

fs.writeFileSync(schemaPath, content, 'utf8');
console.log("Schema converted to SQLite successfully!");
